alf/bin/train_play_test.py

```diff
@@ -289,18 +292,22 @@ def _test_train(self, gin_file, extra_params, root_dir):
         cmd.extend(extra_params or [])
         run_cmd(cmd=cmd, cwd=examples_dir)
 
-    def _test_play(self, root_dir, extra_params):
+    def _test_play(self, root_dir, extra_params, test_video_recording):
         """Test if it can play successfully using configuration and checkpoints
         saved in root_dir.
 
         Args:
             root_dir (str): Root directory where configuration and checkpoints are saved
             extra_params (list[str]): extra parameters used for play
+            test_video_recording (bool): if True, also test if a video can be
+                recorded.
         """
         cmd = [
             'python3', '-m', 'alf.bin.play',
             '--root_dir=%s' % root_dir, '--num_episodes=1'
         ]
+        if test_video_recording:
+            cmd.append('--record_file=%s/play.mp4' % root_dir)
```

Comment:
after run_cmd, need to check
`
file = root_dir + "/play.mp4"
assert os.path.isfile(file) and os.stat(file).st_size > 100`

Could you confirm that the test breaks if the fix is disabled, and the test passes after fix?

Thanks,
Le

Reply:
I'm not sure what you mean here? I've tested locally that I will get an mp4 file after the play. For your frame size inconsistency error, I don't know the reason yet. 

In the current train_play_test.py, if any error happens during the rendering, then the test will fail. No need to check the resulting file.